tests/Subscriber/ConsoleSubscriberTest.php

```diff
@@ -48,7 +52,7 @@ public function testOnPreCreateAstMapEventWithVerboseVerbosity(): void
         $subscriber = new ConsoleSubscriber($output);
         $subscriber->onPreCreateAstMapEvent(new PreCreateAstMapEvent(9999999));
 
-        self::assertSame("Start to create an AstMap for 9999999 Files.\n", $symfonyOutput->fetch());
+        self::assertEquals(new EolAgnosticString("Start to create an AstMap for 9999999 Files.\n"), $symfonyOutput->fetch());
```

Comment:
Wouldn't a search & replace of `\n"` with `".PHP_EOL` work just as well, in some tests? If so, I would prefer it over using `new EolAgnosticString()` in cases like this

Reply:
It would work in some places but not all. Depending on whether the newline is produced by e.g. symfony classes or graphiz classes, or by ones from deptrac different newlines get rendered.

we can improve the current status and get rid of some This asserts in followups IMO.

as can be seen with https://github.com/qossmic/deptrac/issues/756 the PR as is already provides good enough coverage for cross os problem